rsync_tmbackup.sh

```diff
@@ -613,9 +613,10 @@ while : ; do
 	# -----------------------------------------------------------------------------
 	# Add symlink to last backup
 	# -----------------------------------------------------------------------------
-
-	fn_rm_file "$DEST_FOLDER/latest"
-	fn_ln "$(basename -- "$DEST")" "$DEST_FOLDER/latest"
+  if [ "$EXIT_CODE" = 0 ]; then
+    fn_rm_file "$DEST_FOLDER/latest"
+    fn_ln "$(basename -- "$DEST")" "$DEST_FOLDER/latest"
+	fi
```

Comment:
That looks good, but please use tabs for indentation like the rest of the file

Reply:
Thanks for the quick review.  I've updated by using tabs.

Reply:
I think there is one more thing to consider is whether the partial backup should be deleted or not. I would opt for the deletion.

Reply:
I don't think it should because if I'm not mistaken if the backup failed, it will resume from that folder on next run

Reply:
You are right that the partial backup is used as the base for the next run.   Please merge if ok.

Reply:
I added one more commit: I believe the `.inprogress` file can only be removed when rsync succeeded. Otherwise, the previous partial backup will be considered as a full & succeeded backup.

Reply:
I added one more commit to fix a few old indentation errors. Hopefully you do not mind mixing this with the bug fix above :)

Reply:
I rebased on your updated master branch.  Fixed a few more old indentation errors. Hopefully you do not mind mixing this with the bug fix above :)